lib/config/specification.yml

```diff
@@ -1119,6 +1119,9 @@ workspaces:
     - posts_require_privacy_decision
     - stories_are_fixed_fee_by_default
     - currency
+    - target_margin
+    - approver_id
+    - change_orders_enabled
```

Comment:
Sometimes I think we need to re-write this gem so that it doesn't need to know about all the fields ahead of time.

mavenlink-js / backbone models could be a good model for how this gem should work.

Reply:
@ahuth At least for the standard attributes, you don't need them in the specification. You just have to access them as a hash `[attr]` not as a method, `.attr`. 

The reason we've been doing a lot of updates here is because we use this specification for attribute validation for our flatfile export/import and data-loader project. So having this specification really helps us avoid having to manually update lists in two different projects.

Maybe at some point we can have API endpoints on Mavenlink which returns attrs/create_attrs/update_attrs for each object.